### DISCUSSION TITLE
Starting a new set of plots while other plots are running

### DISCUSSION THREAD

**Discussion by cpereyra5199** [Chia-Network/chia-blockchain#5530] [Q&A]
I have a question about concurrent plots.  My system currently supports 10 plots concurrently.  From what I am reading, which I may be understanding incorrectly, it seems after a certain stage I should be able to start spinning up new plots.  Can anyone confirm if this is the case?   For example I currently have 10 plots at about 77% complete and I have started another 5 plots.  They seem to be running fine, but I don't understand how that is possible if I have 20 threads and 15 plots would use 30.  Any explanation would be much appreciated!

Thank you.

**Comment by ilyakooo0** [comment]
What do you mean by "My system currently supports 10 plots concurrently". How did calculate how many plots your machine supports?

The main concern is that running a lot of plots at the same time might starve some resource. For example, if you start 20 plots at the same time, they might all start writing large amounts of data to disk at the same time, and disks might not be able to keep up. If you stagger when plots are started (for example leaving 30 minutes between starts), then it is a lot less likely that all plots will be doing the same operation at the same time – less likely to starve resources.

**Reply by cpereyra5199** [reply]
With my setup, my bottleneck is my CPU which only has 20 threads, at 2 threads per plot, that gives me 10.  So the starving resource in my case would be the CPU.  My M2 and and RAM can both support a lot more than that.  For staggering, in your example of 30 minutes, is that 30 minutes per plot?  So say I create 10 plots concurrently, it would take a total of 300 minutes to start all of them?

**Reply by ilyakooo0** [reply]
> So the starving resource in my case would be the CPU.

I wouldn't be so sure. The only real way to test it would be to actually run the plots and monitor resource usage.

> So say I create 10 plots concurrently, it would take a total of 300 minutes to start all of them?

yes

```
plot 1 ##############...
plot 2    ###########...
plot 3       ########...
...
```
